package.json

```diff
@@ -6,6 +6,8 @@
   "scripts": {
     "prewatch": "npm run clean",
     "watch": "parcel src/index.html --open --no-autoinstall --port 8080",
+    "watch:sassprefixer": "node-sass --output-style expanded --source-map true -o css sass --watch && onchange -d 100 (ścieżka do plików css) -- autoprefixer-cli -o (plik prefixed css)",
+    "watch:browsersync": "node_modules/.bin/browser-sync start --server --files (pliki css i html)",
```

Comment:
skrypt nie może zawierać takiego komentarza pliki css i html i linia wyżej jest to samo